### DISCUSSION TITLE
stable-diffusion-webui-assets' does not exist

### DISCUSSION THREAD

**Discussion by dymension** [TheLastBen/fast-stable-diffusion#2769] [Q&A] [answered]
Hello Ben

thanks for the great work you're doing with this Collab notebook
I'm using it a lot, and it's only because of your work. So thanks

You must be aware of the problems since the new Python update on Google Collab, and you solved it partly, but remains an error which is

**RuntimeError: Directory '/content/gdrive/MyDrive/sd/stable-diffusion-webui/repositories/stable-diffusion-webui-assets' does not exist**

Ful error below :
```
Running on public URL: https://592f7c1e8ba57cfc01.gradio.live/
✔ Connected
Traceback (most recent call last):
  File "/content/gdrive/MyDrive/sd/stable-diffusion-webui/webui.py", line 162, in <module>
    webui()
  File "/content/gdrive/MyDrive/sd/stable-diffusion-webui/webui.py", line 109, in webui
    ui.setup_ui_api(app)
  File "/content/gdrive/MyDrive/sd/stable-diffusion-webui/modules/ui.py", line 1242, in setup_ui_api
    app.mount("/webui-assets", fastapi.staticfiles.StaticFiles(directory=launch_utils.repo_dir('stable-diffusion-webui-assets')), name="webui-assets")
  File "/usr/local/lib/python3.10/dist-packages/starlette/staticfiles.py", line 57, in __init__
    raise RuntimeError(f"Directory '{directory}' does not exist")
RuntimeError: Directory '/content/gdrive/MyDrive/sd/stable-diffusion-webui/repositories/stable-diffusion-webui-assets' does not exist
```

it seems to be when laoding the models

Will you be able to solve that ?

**Comment by dymension** [comment, marked as answer]
I could notice that you updated "fast_stable_diffusion_AUTOMATIC1111.ipynb" this evening
Thanks a lot, I copied your Collab to my gDrive, and it's working fine now

You're the best thanks Ben

If there is a way we can TIP you, please put a link, you deserve it

**Reply by TheLastBen** [reply]
I appreciate it, your appreciation message is enough.
